test/tools/libtesteth/ImportTest.cpp

```diff
@@ -219,10 +219,23 @@ std::tuple<eth::State, ImportTest::ExecOutput, eth::ChangeLog> ImportTest::execu
 	assert(m_envInfo);
 
 	State initialState = _preState;
+	ExecOutput out(std::make_pair(eth::ExecutionResult(), eth::TransactionReceipt(h256(), u256(), eth::LogEntries())));
 	try
 	{
 		unique_ptr<SealEngineFace> se(ChainParams(genesisInfo(_sealEngineNetwork)).createSealEngine());
-		ExecOutput out = initialState.execute(_env, *se.get(), _tr, Permanence::Uncommitted);
+		if (Options::get().jsontrace)
+		{
+			Json::Value trace;
+			StandardTrace st;
+			st.setShowMnemonics();
+			st.setOptions(Options::get().jsontraceOptions);
+			out = initialState.execute(_env, *se.get(), _tr, Permanence::Uncommitted, st.onOp());
+			Json::Reader().parse(st.json(), trace);
```

Comment:
It's preferable to create a function that returns `Json::Value` and just call that from here and the original location.

@winsvega are you in a hurry to get this PR through?